src/ugeneui/src/project_support/ProjectImpl.cpp

```diff
@@ -71,6 +71,23 @@ void ProjectImpl::makeClean() {
     assert(!isTreeItemModified());
 }
 
+void ProjectImpl::sl_removeAllrelationsWithObject(GObject* obj) {
+    QList<GObject*> allObjs;
+    for (Document *d : getDocuments()) {
```

Comment:
Use qAsConst()

Reply:
Fix method name to use correct case for words

Reply:
It is already constant

Reply:
wrap getDocuments() with qAsConst() so there is a guarantee that the collection is not changed during the loop.